src/backend/lima.ts

```diff
@@ -416,7 +416,7 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   get ipAddress(): Promise<string | undefined> {
     return (async() => {
       // Get the routing map structure
-      const state = await this.limaWithCapture('shell', '--workdir=.', MACHINE_NAME, 'cat', '/proc/net/fib_trie');
+      const state = await this.execCommand({ capture: true }, 'cat', '/proc/net/fib_trie');
```

Comment:
I'm a bit confused why you replace `limaWithCapture` with `execCommand` in a few places, but left `limaWithCapture` in others (although it makes sense to have calls to `limaWithCapture` inside `execCommand`.

Reply:
thx